test/functional/test_framework/wallet.py

```diff
@@ -182,6 +182,17 @@ def sendrawtransaction(self, *, from_node, tx_hex):
         from_node.sendrawtransaction(tx_hex)
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
 
+    def send_large_transactions(self, node, array_of_large_tx, num_txns, fee_rate):
+        """Creates and sends large transactions with the passed fee rate."""
+        # Create large transactions by appending txouts in vout
+        for _ in range(num_txns):
+            # Create a self transfer here to get the tx details and then append the vout to increase the tx size
+            tx = self.create_self_transfer(from_node=node, fee_rate=fee_rate)['tx']
+            for txout in array_of_large_tx:
+                tx.vout.append(txout)
+            self.sendrawtransaction(from_node=node, tx_hex=tx.serialize().hex())
+        return num_txns
```

Comment:
is there a reason you have this helper function returning the same variable that was passed in? personally, i find this confusing and would not expect to have to handle a returned value if i were using this in another test

Reply:
Makes sense, as pointed out by other reviewers too, I have taken this into consideration, and converted this into a void function.